template/src/css/main.css

```diff
@@ -54,7 +54,8 @@ input[type="date"] {
     url('./glyphicons-halflings-regular.svg#glyphicons-halflingsregular') format('svg');
 }
 
-body {
+html, body {
+  overflow-x: hidden;
```

Comment:
add a comment to explain why this is necessary

Reply:
Please check the comment.

Reply:
The comment seems incorrect, what you are adding here is `overflow-x`, so add the comment on top of that line. Also, it is probably safe to remove the `body` if you use `html`.